README.md

```diff
@@ -34,3 +34,16 @@ Send a new PR to this repository on GitHub, append your application table. Radxa
 | Project Software: | NixOS |
 | Other Notes:      | - |
 
+| Project Name:     | Lemuria  |
+| ----------------- | -------- |
+| CM3 Model:        | WiFi, eMMC,  RAM >= 4GB    |
+| Project Hardware: | Currently the RPi CM4     |
+| Project Software: | Raspbian/Linux     |
```

Comment:
Not really, we are using Yocto from here: https://gitlab.com/linumiz/lemuria/meta-lemuria

Let me make the repo public access